org.alloytools.alloy.application/bnd.bnd

```diff
@@ -9,6 +11,7 @@
 	org.sat4j.core, \
 	org.sat4j.maxsat, \
 	org.sat4j.pb, \
+	com.io7m.jpplib.io7m-jpplib-core;version='0.7.4',\
```

Comment:
Why do we need to add this? @RuomeiYan

Reply:
See Discord chat history, Aidan is getting another` java.lang.NoClassDefFoundError: de/uka/ilkd/pp/Backend` error. I cannot reproduce the error in my local develop environment, so I did what you did last time by looking up which package it belongs to and importing it in the bnd file. Could you help him out since you have more expertise on the build issues?